### DISCUSSION TITLE
Why the Ipopt/contrib/MatlabInterface folder disappeared after version 3.12?

### DISCUSSION THREAD

**Discussion by 0xF0F0F0F** [coin-or/Ipopt#543] [Q&A] [answered]
Why the Ipopt/contrib/MatlabInterface folder disappeared after version 3.12? Does IPOPT no longer support MATLAB interface?

**Comment by svigerske** [comment, marked as answer]
See CHANGELOG:

https://github.com/coin-or/Ipopt/blob/7447c8008c5e8f5b0489461cd2ff14d692c03520/ChangeLog.md#L349-L351
